src/components/Provider/ProviderHeader/ProviderDeleteButton/ProviderDeleteButton.js

```diff
@@ -39,11 +39,13 @@ const ProviderDeleteButton = ({ provider, disabled }) => {
   const [confirmConfig, setConfirmConfig] = useState(initConfirmConfig);
 
   const redirect = () => {
-    if (provider.isCopyTrading) {
-      navigate("/copyTraders");
-    } else {
-      navigate("/signalProviders");
-    }
+    const marketplaceLink =
+      provider.isCopyTrading && !provider.profitSharing
+        ? "/copyTraders"
+        : provider.isCopyTrading && provider.profitSharing
+        ? "/profitSharing"
+        : "/signalProviders";
```

Comment:
This can be simplified I think
```
provider.isCopyTrading
  ? !provider.profitSharing
    ? "/copyTraders"
    : "/profitSharing"
  : "/signalProviders";
```

Reply:
Done